parachain/runtime/interlay/src/lib.rs

```diff
@@ -394,6 +395,7 @@ impl pallet_transaction_payment::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
     type RuntimeCall = RuntimeCall;
     type RuntimeEvent = RuntimeEvent;
+    type WeightInfo = ();
```

Comment:
Needs benchmarks, probably not a blocker can create a follow-up ticket

Reply:
issue #1185 will resolve it